Makefile

```diff
@@ -49,6 +49,17 @@ GO_OPERATOR_IMAGE_LDFLAGS="-X 'github.com/cilium/tetragon/pkg/version.Version=$(
 GOLANGCILINT_WANT_VERSION = 1.52.2
 GOLANGCILINT_VERSION = $(shell golangci-lint version 2>/dev/null)
 
+# Do a parallel build with multiple jobs, based on the number of CPUs online
+# in this system: 'make -j8' on a 8-CPU system, etc.
+#
+# (To override it, run 'make JOBS=1' and similar.)
+#
+ifeq ($(JOBS),)
+  JOBS := $(shell (getconf _NPROCESSORS_ONLN || grep -E -c '^processor|^CPU[0-9]' /proc/cpuinfo) 2>/dev/null)
```

Comment:
What's the difference with nproc?

Reply:
mostly that I had no idea nproc existed ;-) seems like better choice, I'll update the PR, thanks

Reply:
Oh that's nice then!! Cool :) thanks Jiri

Reply:
You can directly -j $(nproc) then 😺

Reply:
I still want a way to switch it off with JOBS=1 which is handy when you debug build failure
or you want to spread the load differently